### PR TITLE
Table count API added

### PR DESCRIPTION
**Pull request by munkh-altai**
Request example

curl --location --request GET 'https://key-submission.covidmongolia.app/api/count' \
--header 'Authorization: Bearer $token'

Response example {
"CountDiagnosisKeys": 0,
"CountEncryptionKeys": 1,
"CountEvents": 3,
"CountFailedKeyClaimAttempts": 0,
"CountTekUploadCount": 0
}

### CODE REVIEW

File: pkg/server/keyclaim.go [hunk 1]
```diff
@@ -210,3 +212,58 @@ func getIP(r *http.Request) string {
...
+	CountEvents, _ := s.db.CountEvents()
+	CountFailedKeyClaimAttempts, _ := s.db.CountFailedKeyClaimAttempts()
+	CountTekUploadCount, _ := s.db.CountTekUploadCount()
+	data := map[string]int64{}
+
+	data["CountDiagnosisKeys"] = CountDiagnosisKeys
+	data["CountEncryptionKeys"] = CountEncryptionKeys
+	data["CountEvents"] = CountEvents
+	data["CountFailedKeyClaimAttempts"] = CountFailedKeyClaimAttempts
+	data["CountTekUploadCount"] = CountTekUploadCount
```
Comment (amarbayar) [inline pkg/server/keyclaim.go:259]: DB taldaa aldaa zaaval `-1, err` gesen response butsaaj baigaa yum baina. `err` gedgiig ni ter chigeer ni ene dictionary ruu onooj uguhuur json.Marshal hiij chadah bolovuu esvel `err` ni json.Marshal hiihed butetsiin zuruu garaad aldaa garaad yu ch butsaaj chadahaa bolichih bolu? Defensive baidlaar uursduu yag json boldog err butsaaj, onooj ugvul zugeer bolov uu.

**Comment by amarbayar** [comment]
Thanks! @munkh-altai  -- CI test iig ni haraad uguh bolomj baina uu? Shineer nemsen method uudad test baihgui bolhoor fail-dsen baih shig bna. https://github.com/covid-tracing-mongolia/backend-server/pull/6/checks?check_run_id=1469416742

File: pkg/server/keyclaim.go [hunk 2]
```diff
@@ -210,3 +212,58 @@ func getIP(r *http.Request) string {
...
+	hdr := r.Header.Get("Authorization")
+	_, _, ok := s.auth.RegionFromAuthHeader(hdr)
+	if !ok {
+		log(ctx, nil).WithField("header", hdr).Info("bad auth header")
+		http.Error(w, "unauthorized", http.StatusUnauthorized)
+		return result{}
+	}
+
+
+	CountDiagnosisKeys, _ := s.db.CountDiagnosisKeys()
```
Comment (s4kh) [inline pkg/server/keyclaim.go:248]: end hervee err bwal default butsaah uu?
```
CountDiagnosisKeys, err := s.db.CountDiagnosisKeys()
if err != nil {
  log(ctx, err).Error('Sadly error occurred')
  CountDiagnosisKeys = 0 // something like this
}
```

Comment (s4kh) [reply pkg/server/keyclaim.go:248]: Busad endpoint deeree ingej shiidsen yum bna
```
if err != nil {
  log(ctx, err).Error("error counting db values")
  http.Error(w, "server error", http.StatusInternalServerError)
  return
}
```

**Review by s4kh — CHANGES_REQUESTED** [review]

File: pkg/server/keyclaim.go (hunk 1 above)
Comment (s4kh) [reply pkg/server/keyclaim.go:259]: 2 value return hiij bgaa bolohoor aldaa garwal utga ni -1 bolood json marshal deer bol aldaa garahgui bh.
`CountDiagnosisKeys, _ := s.db.CountDiagnosisKeys()` awch bgaa bolohoor err-oo( `_` ni err) eruusuu ashiglahgui bgaa.

**Pull request merged**